### ISSUE TITLE
Static website with docs is gone

### ISSUE THREAD

**Issue by NikolayActionEngine** [uber-web/ocular#308]
Website https://uber-web.github.io/ which was available until very recently is gone now. All links are pointing nowhere (e.g. https://uber-web.github.io/docs/dev-tools/cli/ocular-test). 

I'm not sure if this is a right place for this to be opened, so I used a wrong GitHub project, feel free to point me to the right one and close this ticket.

**Comment by kylebarron** [comment]
I agree it's weird. Links still work if you prefix with `ocular/`, though then they redirect to a page without `ocular/`.

I.e. 
```
https://uber-web.github.io/ocular/docs/dev-tools/configuring-tests
```
works for me, though then it redirects to 
```
https://uber-web.github.io/docs/dev-tools/configuring-tests
```
so if I reload the page it fails.

**Comment by ibgreen** [comment]
Github pages site without their own should be built with the `gatsby --prefix-paths` option (check exact spelling), maybe that was not done?

**Comment by kylebarron** [comment]
I think that's the most likely conclusion.

**Comment by ibgreen** [comment]
Should be part of the website `package.json` in this repo. There is also a PREFIX_PATHS option that needs to go into the gatsby config. Maybe that needs to be done at the top level now and no longer in the ocular theme plugin config?

**Comment by ibgreen** [comment]
We should also check probe.gl - another repo that does not have its own domain.

**Comment by kylebarron** [comment]
It is included in the scripts
https://github.com/uber-web/ocular/blob/143f1bdd4414290e8ab1d8b2834affe930ba7053/website/package.json#L12-L13

**Comment by kylebarron** [comment]
Probe gl seems to work fine, except the top-left "Home" link is wrong. But I'm guessing that's a separate issue.
